pallets/unified-accounts/src/lib.rs

```diff
@@ -256,6 +275,11 @@ impl<T: Config> Pallet<T> {
         });
         Ok(evm_address)
     }
+
+    /// Charge the (exact) storage fee (polietly) from the user and burn it
+    fn charge_storage_fee(who: &T::AccountId) -> Result<BalanceOf<T>, DispatchError> {
+        T::Currency::burn_from(who, T::AccountMappingStorageFee::get(), Exact, Polite)
+    }
```

Comment:
Why not just call this directly? 🤷‍♂️

Reply:
I could, it's just one line.
But since we are using this in two places, I wanted to make sure (and in future) it has same args in currency call, like `Exact` & `Polite`.

I'm fine with inlining this if there is strong opinion against it.